### DISCUSSION TITLE
更新日志 - 22.03.5 - 20231124

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1134] [Announcements]
# 系统
1. 更新 Mediatek 系列无线网卡驱动，并支持 MT7922。（如果第一次启动没出现无线菜单，就重启一次试试）
2. Intel AX201 增加新fw文件，某些版本需要
3. 内核支持 Quectel RM500U-CN 5G 模块（需要自行配置，可能需要使用特定拨号工具）

# 设备
* x86：
  1. EFI 未压缩镜像对齐到 4KB，方便直接解压固件给虚拟机使用（可能对性能有好处）
* rk35xx：
  1. 修复 R6S/R6C 的 USB 3.0 接口 #1128
  2. 修复 H28K 的 PPPoE 拨号性能 #1125
  3. H69K 优化风扇调度
  4. uboot 修改长按电源键 pmic 强制关机的触发时间，从6秒改成10秒，只对重新刷机有效
  5. ffmpeg 支持 mjpeg 硬解，支持USB摄像头数据硬件转码

**Comment by Mr-Fook** [comment]
啥时候可以下载阿

**Comment by lbsnake** [comment]
X86机器无法更新这版EFI固件，无论在线还是手动刷写都无法更新，即便选择不保留配置也不行，重启完系统后还是上一版20231027固件。

**Reply by KingKeepcalm** [reply]
昨天在线升级没有问题。

**Reply by wanyuxi88** [reply]
我也无法升级，在线和本地都不行

**Reply by lbsnake** [reply]
是否用的也是X86EFI固件？

**Reply by ZEROKotori** [reply]
我也更新不了，卡在正在更新界面，用的是X86EFI固件

**Comment by murraychou** [comment]
H69K的更新包好像还没上传

**Comment by noahyann** [comment]
rtl的 8852BE 驱动会有吗? https://github.com/HRex39/rtl8852be/

**Comment by w4575421** [comment]
有没有支持JDC的计划?

**Comment by PigsHeo** [comment]
请问什么时候可以支持 lsi sas2208 的raid卡驱动

**Reply by jjm2473** [reply]
前几次更新不是支持了吗 https://github.com/istoreos/istoreos/discussions/1081

**Reply by PigsHeo** [reply]
rh1288v3上的2208raid，不能读取磁盘信息

**Reply by PigsHeo** [reply]
![61e3bf20cebe70d3f5b7cdc42bd8bd1](https://github.com/istoreos/istoreos/assets/854552/9da2087a-b4d8-46b4-bc44-1b37f30ff66a)

**Reply by PigsHeo** [reply]
2208好像没有

**Reply by jjm2473** [reply]
lspci -nnk 看看完整的列表

**Comment by weiqincc** [comment]
有没有兄弟反馈一下，esxi 装虚拟机 这版支持N100 核显直通了没，可以我就从测试版升级了。

**Comment by AaronHzy** [comment]
更新后DNS错误什么情况，手动改DNS也不行

**Reply by 15270637456** [reply]
我也是，最近半年一直出现这个问题

**Comment by bobotm** [comment]
修修 UPNP吧， 没法用，好久了这个问题。

**Reply by jjm2473** [reply]
取消勾选强制转发试试啊

**Comment by lwj1994** [comment]
iStoreOS 22.03.5 2023112413      升级后，磁盘出现各种 问题，先是磁盘只读无法写入，格式化后，又出现了磁盘无法挂载，格式化后直接识别不出磁盘了 ......

**Reply by jjm2473** [reply]
可能硬盘坏了

**Reply by lwj1994** [reply]
不太像，升级之前好好的。放在那一直没动

**Reply by jjm2473** [reply]
那降级固件试试，估计还是硬件问题

**Comment by wanyuxi88** [comment]
我的x86还是无法升级，在线和本地都升级失败

发自我的iPhone

**Reply by jjm2473** [reply]
接了屏幕的话，升级过程中用手机录像记录下，看看有没有什么错误
